rf-feature-list.md

```diff
@@ -0,0 +1,35 @@
+|---------------------------------|--------------------------------------------------------------------|----------------|
+| <b>Moment features</b>          |                                                                    |                |
+| avg                             | Mean                                                               | g              |
+| std                             | Standard deviation                                                 | g              |
+| skew                            | Skewness                                                           |                |
+| kurt                            | Kurtosis                                                           |                |
+| <b>Quantile Features</b>        |                                                                    |                |
+| min                             | Minimum                                                            | g              |
+| q25                             | Lower quartile                                                     | g              |
+| med                             | Median                                                             | g              |
+| q75                             | Upper quartile                                                     | g              |
+| max                             | Maximum                                                            | g              |
+| <b>Autocorrelation features</b> |                                                                    |                |
+| acf_1st_max                     | Maximum autocorrelation                                            |                |
+| acf_1st_max_loc                 | Location of 1st autocorrelation maximum                            | s              |
+| acf_1st_min                     | Minimum autocorrelation                                            |                |
+| acf_1st_min_loc                 | Location of 1st autocorrelation minimum                            | s              |
+| acf_zeros                       | Number of autocorrelation zero-crossings                           |                |
+| <b>Spectral features</b>        |                                                                    |                |
+| pentropy                        | Spectral entropy calculated from the power spectrum                |                |
+| power                           | Sum of the power spectrum                                          |                |
+| f1, f2, f3                      | Dominant frequencies in the power spectrum                         | Hz             |
+| p1, p2, p3                      | Power associated with respective dominant frequencies              |                |
+| <b>FFT features</b>             |                                                                    |                |
```

Comment:
Let's drop this row (**FFT features**) so that fft0, fft1, etc. are under **Spectral features** 
<img width="626" alt="image" src="https://github.com/OxWearables/stepcount/assets/5551642/d824c5f7-36ff-411d-a7c5-e4f47120d992">